client/src/main/java/com/flipkart/flux/client/runtime/FluxHttpClient.java

```diff
@@ -0,0 +1,6 @@
+package com.flipkart.flux.client.runtime;
+
+public interface FluxHttpClient {
+
+    void postOverHttp(Object dataToPost, String pathSuffix);
```

Comment:
What we can do is, add another method which takes the type to be returned as parameter (or can be done via generics as well), and use object mapper to convert the response.